cql3/query_processor.cc

```diff
@@ -27,6 +27,7 @@
 #include "utils/hashers.hh"
 #include "utils/error_injection.hh"
 #include "service/migration_manager.hh"
+#include "service/storage_service.hh"
```

Comment:
You should be able to avoid this include by creating a wrapper e.g. in migration_manager. Migration manager should become a simple proxy between CQL and the data dictioanary/DDL implementation in the future.
@tgrabiec wdyt?

Reply:
I've replaced it with a fwd declaration instead.